onnxruntime/core/providers/cpu/tensor/upsample.cc

```diff
@@ -494,6 +496,12 @@ void ResizeBiCubic(
     }
   }
 
+  // setup up temp arrays to hold coefficients when exclude_outside is set to true
+  std::array<float, CubicModeGridLength> y_coeff_holder;
```

Comment:
>CubicModeGridLength [](start = 20, length = 19)

This constant was previously used for maps that allocate data on the heap. How big is this constant here that you allocate to arrays on the stack? Can it get big enough that we might have a problem?

Reply:
No - for cubic mode the grid length that we use is always 4 ...